site/docs/v1/tech/apis/_user-comment-search-request-parameters.adoc

```diff
@@ -0,0 +1,42 @@
+
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}comment# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the User Comment text. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `comment` field.
+
+[field]#{parameter_prefix}numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#{parameter_prefix}orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `name ASC`#::
+The field to order the search results as well as an order direction.
++
+The possible values are:
++
+ * `comment` - the User Comment text
```

Comment:
hmmm. this seems weird to order by. But I guess if you have an automated comment integration...

Reply:
I could see wanting to sort by:

* commenterId
* userId
* insertInstant
* tenantId

I think it is fine to leave sorting by comment in, because it depends on how structured the comment is, and you've already written the code.

If the comment is user input, then sorting by it is probably not useful. If it is on the other hand machine generated, it might be useful to be able to sort on it.